### PR TITLE
Add completion files for OpenShift/K8s tools

### PR DESCRIPTION
**Pull request by ifireball**
Including: crc, oc, tkn, helm

### CODE REVIEW

File: completions/crc.completion.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#! bash oh-my-bash.module
+if _omb_util_command_exists crc; then
+  source <(crc completion bash)
```
Comment (akinomyoga) [inline completions/crc.completion.sh:3]: OMB also supports bash 3.2, but bash 3.2 doesn't allow sourcing from pipes. Could you intead use `eval -- "$(xxx completion bash)"`?

File: completions/helm.completion.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#! bash oh-my-bash.module
+if _omb_util_command_exists helm; then
+  source <(helm completion bash)
+fi
+
```
Comment (akinomyoga) [inline completions/helm.completion.sh:5]: Could you strip empty lines at the end of files?

**Review by akinomyoga — CHANGES_REQUESTED** [review]
Thank you for your contributions. Could you check these comments?

File: completions/crc.completion.sh (hunk 1 above)
Comment (ifireball) [reply completions/crc.completion.sh]: done

File: completions/helm.completion.sh (hunk 2 above)
Comment (ifireball) [reply completions/helm.completion.sh]: done

File: completions/oc.completion.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#! bash oh-my-bash.module
+if _omb_util_command_exists oc; then
+  eval -- "$(oc completion bash)"
+fi
+
```
Comment (akinomyoga) [inline completions/oc.completion.sh:5]: Thanks for updating the PR! and also this file.

Comment (ifireball) [reply completions/oc.completion.sh]: Whoops... done.

**Review by akinomyoga — APPROVED** [review]

File: completions/oc.completion.sh (hunk 3 above)
Comment (akinomyoga) [reply completions/oc.completion.sh]: Thanks!

**Comment by akinomyoga** [comment]
Thank you for your quick responses! I think I'll merge this PR after waiting for a few days. Thanks!

**Comment by akinomyoga** [comment]
Thanks. I've merged the PR.

**Pull request merged**